chainntnfs/btcdnotify/btcd.go

```diff
@@ -566,7 +566,7 @@ func (b *BtcdNotifier) confDetailsFromTxIndex(txid *chainhash.Hash,
 	// then we may be able to dispatch it immediately.
 	tx, err := b.chainConn.GetRawTransactionVerbose(txid)
 	if err != nil {
-		// If the transaction lookup was successful, but it wasn't found
+		// If the transaction lookup was succesful, but it wasn't found
```

Comment:
_googles "succesful_. Did you mean: "successful"?

Reply:
fixed